feature/profilecard/src/iosMain/kotlin/io/github/droidkaigi/confsched/profilecard/ProfileCardViewController.kt

```diff
@@ -22,6 +24,10 @@ fun profileCardViewController(
                 imageBitmap.toUiImage() ?: UIImage(),
             )
         },
+        // FIXME This is a workaround. For permanent support, we will get the inset value etc. from the iOS side and respond.
+        contentPadding = PaddingValues(
+            bottom = 30.dp,
```

Comment:
Now, I understand that we don't exceed the height of the bottom tab bar, right? Could we add a comment like this?

```suggestion
            bottom = 30.dp, // Hight of bottom tab bar
```